packages/appstore/src/components/app.scss

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@700&display=swap');
```

Comment:
@sergei-deriv why do we need ubuntu font?

Reply:
@shahzaib-deriv you have to ask design team about this :) I just followed the design from figma:

<img width="839" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/5c33c990-c1dd-4930-82d6-e533c11f8c98">

Reply:
@shahzaib-deriv hope it's ok now and my reply is good for you :) I'll resolve this conversation, but you are free to create one more)